docs/UserGuide.md

```diff
@@ -141,34 +141,44 @@
 
 Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### Adding a floating task : `add`
+### Adding a floating task : `task`
 
 Adds a floating task to TaskTracker.
 
-    add "FLOATING_TASK_NAME" [PRIORITY]
+    task "FLOATING_TASK_NAME" [PRIORITY]
 
-* Task name should be in a pair of quotation marks.
+* Task name should be in a pair of quotation marks. And quotations marks are
+not allowed in task name.
 
-* The floating task will be stored and shown to user according to the order of
-  the time they created.
+* Task name could be a single word or a phrase, white space are allowed.
+
+* The floating task will be shown according to their `PRIORITY`.
+
+* Two floating tasks with the same `PRIORITY` will be shown according to
+  the order of the time they created.
 
 * The `PRIORITY` attribute is an integer which ranges from `0` to `5`.
 
 * The default piority of the floating task is 0.
 
 #### Examples
 
-* `add EE2020 "lab report" 5`
+* `task "EE2020 lab report" 5`
+
+To create a task called `EE2020 lab report` with `PRIORITY` of 5.
+
+* `task ProgressReflection`
 
-* `add "ProgressReflection"`
+To create a task called `ProgressReflection` with default `PRIORITY` of 0.
 
-### Adding a event-like task : `add`
+### Adding a event-like task : `event`
 
 Adds an event with specific starting and ending date or time to TaskTracker.
 
-    add “EVENT_NAME” <STARTING_DATE> <STARTING_TIME> <ENDING_DATE> <ENDING_TIME> [loc/LOCATION]
+    event “EVENT_NAME” <STARTING_DATE> <STARTING_TIME> <ENDING_DATE> <ENDING_TIME> [loc-LOCATION]
```

Comment:
@nightism Okay, I've implemented the `event` command in #37 (with the exception of `location`) and it seems to work well, so I think this command design is good enough,